Dockerfile.setup-etcd-environment

```diff
@@ -1,10 +1,8 @@
-FROM golang:1.10.3 AS build-env
-
-COPY . /go/src/github.com/openshift/machine-config-operator
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/machine-config-operator
+COPY . .
 RUN WHAT=setup-etcd-environment ./hack/build-go.sh
 
-FROM openshift/origin-base:v4.0.0
-COPY --from=build-env /go/src/github.com/openshift/machine-config-operator/_output/linux/amd64/setup-etcd-environment /bin/setup-etcd-environment
-
-ENTRYPOINT ["/bin/setup-etcd-environment"]
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+COPY --from=builder /go/src/github.com/openshift/machine-config-operator/_output/linux/amd64/setup-etcd-environment /usr/bin/
+ENTRYPOINT ["/usr/bin/setup-etcd-environment"]
```

Comment:
LGTM overall, only very minor nit is some of the files don't have ending newlines.